pool/pool.go

```diff
@@ -1811,3 +1811,26 @@ func (p *Pool) Close() {
 	p.cancel()
 	<-p.closedCh
 }
+
+// SyncContainerWithNetwork applies network configuration received via
+// the Pool to the container. Changes the container if it does not satisfy
+// network configuration.
+//
+// Pool and container MUST not be nil.
+//
+// Returns any error that does not allow reading configuration
+// from the network.
+func SyncContainerWithNetwork(ctx context.Context, cnr *container.Container, p *Pool) error {
+	ni, err := p.NetworkInfo(ctx)
+	if err != nil {
+		return fmt.Errorf("network info: %w", err)
+	}
+
+	if ni == nil {
```

Comment:
`ni != nil` if `err == nil`, or you think this should be clarified in `Pool.NetworkInfo` docs?

Reply:
> this should be clarified in Pool.NetworkInfo docs?

yes, otherwise i think we could not rely on it

Reply:
https://github.com/nspcc-dev/neofs-sdk-go/issues/291